applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -394,6 +395,99 @@ void BaseSolidElement::CalculateRightHandSide(
     CalculateAll( temp, rRightHandSideVector, rCurrentProcessInfo, CalculateStiffnessMatrixFlag, CalculateResidualVectorFlag );
 }
 
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void BaseSolidElement::AddExplicitContribution(
+    const VectorType& rRHSVector, 
+    const Variable<VectorType>& rRHSVariable,
+    Variable<double>& rDestinationVariable,
+    const ProcessInfo& rCurrentProcessInfo
+    ) 
+{
+    KRATOS_TRY;
+
+    auto& r_geom = this->GetGeometry();
+    const SizeType dimension = r_geom.WorkingSpaceDimension();
+    const SizeType number_of_nodes = r_geom.size();
+    const SizeType element_size = dimension * number_of_nodes;
+
+    // Compiting the nodal mass
+    if (rDestinationVariable == NODAL_MASS ) {
+        Matrix element_mass_matrix = ZeroMatrix(element_size, element_size);
+        ProcessInfo temp_process_information = rCurrentProcessInfo; // Dummy
+        temp_process_information.SetValue(COMPUTE_LUMPED_MASS_MATRIX, true);
+        this->CalculateMassMatrix(element_mass_matrix, temp_process_information);
+
+        for (IndexType i = 0; i < number_of_nodes; ++i) {
+            double aux_nodal_mass = 0.0;
+
+            const IndexType index = i * dimension;
+
+            for (IndexType j = 0; j < element_size; ++j) {
+                aux_nodal_mass += element_mass_matrix(index, j);
+            }
+
+            #pragma omp atomic
+            r_geom[i].GetValue(NODAL_MASS) += aux_nodal_mass;
+        }
+    }
+
+    KRATOS_CATCH("")
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void BaseSolidElement::AddExplicitContribution(
+    const VectorType& rRHSVector, 
+    const Variable<VectorType>& rRHSVariable,
+    Variable<array_1d<double, 3>>& rDestinationVariable,
+    const ProcessInfo& rCurrentProcessInfo
+    ) 
+{
+    KRATOS_TRY;
+
+    auto& r_geom = this->GetGeometry();
+    const auto& r_prop = this->GetProperties();
+    const SizeType dimension = r_geom.WorkingSpaceDimension();
+    const SizeType number_of_nodes = r_geom.size();
+    const SizeType element_size = dimension * number_of_nodes;
+    
+    Vector damping_residual_contribution = ZeroVector(element_size);
+    
+    // Calculate damping contribution to residual -->
+    if (r_prop.Has(RAYLEIGH_ALPHA) || r_prop.Has(RAYLEIGH_BETA)) {
+        Vector current_nodal_velocities = ZeroVector(element_size);
+        this->GetFirstDerivativesVector(current_nodal_velocities);
+        
+        Matrix damping_matrix = ZeroMatrix(element_size, element_size);
+        ProcessInfo temp_process_information = rCurrentProcessInfo; // We can't pass const ProcessInfo
+        temp_process_information.SetValue(COMPUTE_LUMPED_MASS_MATRIX, true);
```

Comment:
there is a variable in StructMech that does the exact opposite: `USE_CONSISTENT_MASS_MATRIX` which is used by the Beams/Shells
=> We have to unify this

Plus this should not be a processinfo-var but one in the properties, otherwise I cannot use different ones for different elements 

Since I am not the expert on what the default (lumped or consistent) should be, can you please open an issue to discuss this with the rest of StructuralMechanics?

Reply:
COMPUTE_LUMPED_MASS_MATRIX is already implemented in the base element, and it was defined long long time ago, the others elements should change, not this one

Reply:
I can open an issue after this, but my position will be in use the oldest and most extended one. COMPUTE_LUMPED_MASS_MATRIX in this case